cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers.go

```diff
@@ -334,8 +338,8 @@ func GetEtcdPeerAltNames(cfg *kubeadmapi.InitConfiguration) (*certutil.AltNames,
 
 	// create AltNames with defaults DNSNames/IPs
 	altNames := &certutil.AltNames{
-		DNSNames: []string{cfg.NodeRegistration.Name, "localhost"},
-		IPs:      []net.IP{advertiseAddress, net.IPv4(127, 0, 0, 1), net.IPv6loopback},
+		DNSNames: []string{cfg.NodeRegistration.Name},
```

Comment:
Why are you removing loopback from the SAN?

Reply:
I'd have to dig through history but I remember us adding it on purpose.

Reply:
sound strange, but restored